test/unstable/Map/parSafe.prediff

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+OUTFILE=$2
+TMPFILE=$OUTFILE.prediff.tmp
+
+# filter out line numbers
+sed -E 's/\.chpl:[0-9]*:/\.chpl:nnnn:/' < $OUTFILE > $TMPFILE
+cat $TMPFILE > $OUTFILE
+
+rm $TMPFILE
```

Comment:
Make sure you add a newline at the end of this file.

(There is a VS code extension called "NewLine" that does this for you automatically whenever you save a file)

Reply:
Also, in my offline message, I was suggesting that you add these `.prediff` files to `types/chplhashtable/noHashForRecordWithEq` and `types/chplhashtable/noHashForRecordWithEq2` to prevent the test failures you were seeing.

You don't need them here since the line numbers are coming from the test itself rather than an internal module file.

Reply:
Ohhh, I see. Thank you for the clarification